### PR TITLE
Release 0.6.2 - Adaptive phase filter documentation

### PR DESCRIPTION
**Pull request by hjkristenson**
This PR adds documentation for the adaptive phase filter, and the options available for adjusting the parameter value when ordering InSAR products on demand.

### CODE REVIEW

**Comment by hjkristenson** [comment]
This PR should not be merged until after the [hyp3-gamma PR](https://github.com/ASFHyP3/hyp3-gamma/pull/506) has been released.

**Review by jtherrmann — DISMISSED** [review]

File: docs/guides/insar_product_guide.md [hunk 1]
```diff
@@ -20,33 +20,58 @@ Users are cautioned to read the sections on [limitations](#limitations "Jump to
...
+!!! tip "New: Adaptive Phase Filter parameter is now customizable!"
 
-    As of September 27, 2022, the water mask used for this option is no longer buffered. The original water mask had a 3 km buffer on coastlines and a 5 km buffer on the shorelines of inland waterbodies. This was to reduce the chance that valid land pixels would be excluded from phase unwrapping, but it appears that the inclusion of more water pixels is more detrimental to phase unwrapping than the exclusion of some land pixels. Visit our [InSAR Water Masking Tutorial](https://storymaps.arcgis.com/stories/485916be1b1d46889aa436794b5633cb "InSAR Water Masking StoryMap" ){target=_blank} for more information.
+    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to both the SDK and Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```
Comment (jtherrmann) [inline docs/guides/insar_product_guide.md:29]:
```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```

**Review by jtherrmann — DISMISSED** [review]

**Comment by jtherrmann** [comment]
Trufflehog can be safely ignored.

File: docs/guides/insar_product_guide.md [hunk 2]
```diff
@@ -20,33 +20,58 @@ Users are cautioned to read the sections on [limitations](#limitations "Jump to
...
+!!! tip "New: Adaptive Phase Filter parameter is now customizable!"
 
-    As of September 27, 2022, the water mask used for this option is no longer buffered. The original water mask had a 3 km buffer on coastlines and a 5 km buffer on the shorelines of inland waterbodies. This was to reduce the chance that valid land pixels would be excluded from phase unwrapping, but it appears that the inclusion of more water pixels is more detrimental to phase unwrapping than the exclusion of some land pixels. Visit our [InSAR Water Masking Tutorial](https://storymaps.arcgis.com/stories/485916be1b1d46889aa436794b5633cb "InSAR Water Masking StoryMap" ){target=_blank} for more information.
+    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```
Comment (hjkristenson) [inline docs/guides/insar_product_guide.md:29]:
```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```

File: docs/guides/insar_product_guide.md [hunk 3]
```diff
@@ -20,33 +20,58 @@ Users are cautioned to read the sections on [limitations](#limitations "Jump to
...
+!!! tip "New: Adaptive Phase Filter parameter is now customizable!"
 
-    As of September 27, 2022, the water mask used for this option is no longer buffered. The original water mask had a 3 km buffer on coastlines and a 5 km buffer on the shorelines of inland waterbodies. This was to reduce the chance that valid land pixels would be excluded from phase unwrapping, but it appears that the inclusion of more water pixels is more detrimental to phase unwrapping than the exclusion of some land pixels. Visit our [InSAR Water Masking Tutorial](https://storymaps.arcgis.com/stories/485916be1b1d46889aa436794b5633cb "InSAR Water Masking StoryMap" ){target=_blank} for more information.
+    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```
Comment (jtherrmann) [inline docs/guides/insar_product_guide.md:29]:
```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, and is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**